azurerm/internal/services/containers/client/client.go

```diff
@@ -2,8 +2,8 @@ package client
 
 import (
 	"github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-10-01/containerinstance"
-	"github.com/Azure/azure-sdk-for-go/services/containerregistry/mgmt/2018-09-01/containerregistry"
 	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2020-03-01/containerservice"
+	"github.com/Azure/azure-sdk-for-go/services/preview/containerregistry/mgmt/2019-12-01-preview/containerregistry"
```

Comment:
[2019-05-01](https://github.com/Azure/azure-sdk-for-go/tree/master/services/containerregistry/mgmt/2019-05-01/containerregistry) appears to be the latest stable API version - can we use that version (or a newer GA version) instead?

Reply:
Since this PR is making preparations of the CMK in ACR, 2019-05-01 does not have CMK related feature. The only api-version that contains CMK is 2019-12-01-preview.

Reply:
so there's been no stable release of this API (e.g. guarantees of no breaking changes) for 16 months (tomorrow)? shouldn't ARM Review be catching that? cc @JeffreyRichter

Reply:
@tombuildsstuff I've raised this internally. The team has promised to reply in a few days.